pages/index.js

```diff
@@ -1,32 +1,125 @@
 import TodoListItem from '../components/TodoListItem'
 import AddTask from '../components/AddTask'
 import { useEffect, useState } from 'react'
-import axios from '../utils/axios'
+import axios from 'axios'
 import { useAuth } from '../context/auth'
+import { API_URL } from '../utils/constants'
+import AuthRequired from '../middlewares/auth_required'
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css'
+import { ToastContainer, toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
 
 export default function Home() {
   const { token } = useAuth()
   const [tasks, setTasks] = useState([])
 
+  function updateToast(){
+    toast.info("Task updated successfully!", {
+      position: "bottom-right",
+    })
+  }
+
+  function deleteToast(){
+    toast.warning("Task deleted successfully!", {
+      position: "bottom-right",
+    })
+  }
+
   function getTasks() {
-    /***
-     * @todo Fetch the tasks created by the user.
-     * @todo Set the tasks state and display them in the using TodoListItem component
-     * The user token can be accessed from the context using useAuth() from /context/auth.js
-     */
+    axios({
+      url: API_URL + 'todo/',
+      method: 'get',
+      headers: {
+        Authorization: 'Token ' + token
+      }
+    }).then(res => {
+      console.log(res.data)
+      setTasks(res.data)
+      console.log("Tasks recieved")
```

Comment:
Use Toast instead of console logging

Reply:
I thought it is not necessary to include a toast for get tasks function, since I've already used toast for add task function.